includes/compat/class-wc-stripe-subscriptions-helper.php

```diff
@@ -54,6 +71,11 @@ public static function get_some_detached_subscriptions() {
 				continue;
 			}
 
+			// Filter subscriptions not renewing in the next month
```

Comment:
Not blocking: if we plan to allow users to pre-select subscriptions, e.g. from the admin, we probably want to separate this filtering step from the logic that checks each subscription. So we'd probably want an `is_subscription_detached( $subscription )` function and a `get_detached_subscription_data( $subscription )` function.

But it may be worth tackling that in the PR where we add that functionality.